modules/mysql/src/main/scala/doobie/mysql/JavaTimeInstances.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2013-2020 Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package doobie.mysql
+
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+import doobie.Meta
+import doobie.enumerated.{JdbcType => JT}
+import doobie.util.meta.MetaConstructors
+
+trait JavaTimeInstances extends MetaConstructors {
+
+  implicit val JavaTimeOffsetDateTimeMeta: Meta[java.time.OffsetDateTime] =
+    Basic.oneObject(
+      JT.Timestamp,
+      List(JT.VarChar, JT.Date, JT.Time),
+      classOf[java.time.OffsetDateTime]
+    )
+
+  implicit val JavaTimeInstantMeta: Meta[java.time.Instant] =
```

Comment:
Shouldn't this be the "primary" Meta instance and the one for `OffsetDateTime` be derived from it?

Reply:
`Instant` isn't mentioned in the JDBC spec so drivers don't have to provide a way to convert a column to/from it. The MySQL driver errors if you ask it for an `Instant` so it is implemented in terms of `OffsetDateTime`.